app/templates/notifications/user-research-consent.html

```diff
@@ -0,0 +1,72 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}User reserch notification – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": dashboard_url,
+        "label": "Your account"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="grid-row notification-user-research">
+
+  <div class="column-two-thirds">
+    {% with heading = "Join the user research mailing list" %}
+      {% include "toolkit/page-heading.html" %}
+    {% endwith %}
+    
+    <p class="text">
+      Help us improve the Digital Marketplace by taking part in user research.
+    </p>
+    <p class="text">
+      You don’t have to take part just because you’ve signed up and you can unsubscribe at any time.
+    </p>
+
+    <form action="{{ url_for('main.user_research_consent') }}" method="POST">
+      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+      {%
+        with
+        type = "checkbox",
+        name = "user_research_opt_in",
+        value = [userResearchOptIn],
```

Comment:
You should be able to get this value from `current_user` which is made available in all templates. That way you can have one less template argument.

Reply:
does this value update on refresh each time? ie. If I opt in, on the next view, will it be true?

Reply:
The Flask user object is populated via an API call on login. If you opt in during the session I don't think it will update the user object until you log out/log back in again.

Reply:
Have checked, there is a small delay before this value is updated.